commit 2761f059008b7734c0918a995b0d9171f4af059f (refs/changes/44/5044/2)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-11-09T12:14:23-08:00 (11 months ago)
    
    TOOLS-945 sdcadm failed to assign platform without reporting failure
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/lib/models/server.js b/lib/models/server.js
index ba59962..ecb351e 100644
--- a/lib/models/server.js
+++ b/lib/models/server.js
@@ -1622,7 +1622,7 @@ ModelServer.prototype.updateBootParams = function (bootParams, callback) {
         });
 
         self.modify(server, function (modifyError) {
-            if (error) {
+            if (modifyError) {
                 callback(new VError(modifyError,
                                     'modifying server boot param'));
                 return;
diff --git a/package.json b/package.json
index 13d33bc..01f2fab 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.13.0",
+  "version": "1.13.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
